experimental/addons/datadog/definitions/datadog.cue

```diff
@@ -76,7 +76,7 @@ template: {
     let sourceAnnotation = {
         if parameter.source != _|_ {
             metadata: annotations: {
-                ("ad.datadoghq.com/"+parameter.serviceName+".logs"): "[{\"source\": \""+parameter.source+"\"}]"
```

Comment:
Do we still need service name?

Reply:
Yes, it's used elsewhere (e.g. lines 18 & 35).

It's used as an alias for the service within datadog ui when presenting data, but it was incorrect to use it in the logging source annotation, it needs to be the container name here.